Presentation: Support displaying relationship properties (#3808)
iTwin/itwinjs-core: docs/presentation/content/RelatedPropertiesSpecification.md

```

![Example of using the "nested related properties" attribute](./media/relatedpropertiesspecification-with-nestedrelatedproperties-attribute.png)

### Attribute: `relationshipProperties`

> List of names or definitions of relationship class properties that should be included in the content. In addition, a couple of special values are allowed:

Reviewer: I think it's important to mention that we're always using the last relationship in the given relationship path in `propertiesSource` (important when the path is more than 1 step long).

Reviewer: Not sure I like the change. For me something like this sounds better:

> List of names or definitions of relationship class properties that should be included in the content. The properties are taken from the relationship used in the last step of the relationship path specified in the [`propertiesSource`](#attribute-propertiessource). 

@roluk, what's your opinion?

Reviewer 2: I agree with your judgement, but I provided a shorter alternative in another comment.

> List of names or definitions of relationship class properties that should be included in the content. The properties are taken from the relationship used in the last step of the relationship path specified in the [`propertiesSource`](#attribute-propertiessource). In addition, a couple of special values are allowed:

Reviewer 2: Could be made shorter:

```suggestion
Lists ECRelationshipClass properties that should be included in the content. Only the properties from the last relationship of [`propertiesSource`](#attribute-propertiessource) path are accessible. In addition, a couple of special values are allowed:
```

|                   |                                                             |
| ----------------- | ----------------------------------------------------------- |
| **Type**          | `Array<string \| PropertySpecification> \| "_none_" \| "*"` |
| **Is Required**   | No                                                          |
| **Default Value** | `[]`                                                        |

```ts
[[include:Presentation.Content.Customization.RelatedPropertiesSpecification.RelationshipProperties.Ruleset]]
```

![Example of using the "relationship properties" attribute](./media/relatedpropertiesspecification-with-relationshipproperties-attribute.png)

### Attribute: `forceCreateRelationshipCategory`

Specifies whether a relationship category should be created regardless of whether any relationship properties were included.

|                   |           |
| ----------------- | --------- |
| **Type**          | `boolean` |
| **Is Required**   | No        |
| **Default Value** | `false`   |

```ts
[[include:Presentation.Content.Customization.RelatedPropertiesSpecification.ForceCreateRelationshipCategory.Ruleset]]
```

> ![Example of using the "force create relationship category" attribute](./media/relatedpropertiesspecification-with-forcecreaterelationshipcategory-attribute.png)

Reviewer 2: It would be easier to understand the effect of `forceCreateRelationshipCategory` if there was an another screenshot where it's set to `false`.
